docs/source/bentocloud/how-tos/manage-models-and-bentos.rst

```diff
@@ -1,97 +1,54 @@
-which enables you to push these local assets to a remote repository
-for efficient management and team collaboration.
-
-Pull and Push Models
+Pull and push models
 ====================
 
-New Model Versions
-------------------
-
-As your ML models evolve, you'll often need to create new versions.
-Here's how you can manage those versions:
-
-1. **Pull a Model:** To pull a model from the repository,
-   use the ``bentoml models pull`` CLI command followed by the model name and tag.
+* **Pull a model**: To fetch a specific version of a model from BentoCloud, run:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   bentoml models pull <model_name>:<tag>
+     bentoml models pull <model_name>:<tag>
 
-1. **Push a Model:** Push a new model version to the repository
-   using the ``bentoml models push`` command followed by the model name and tag.
+* **Push a model**: To upload a new version of an existing model to BentoCloud, run:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   bentoml models push <model_name>:<tag>
+     bentoml models push <model_name>:<tag>
 
-You can view your models in BentoCloud’s UI in `cloud.bentoml.com/models <http://cloud.bentoml.com/models>`_.
+The **Models** page displays all available models on BentoCloud.
 
 .. image:: ../../_static/img/bentocloud/manage-models.gif
    :alt: manage-models.gif
 
-Pull and Push Bento
-===================
-
-New Bento Versions
-------------------
-
-Similarly, you can manage different versions of Bentos:
+Pull and push Bentos
+====================
 
-1. **Pull a Bento:** To pull a specific Bento version from the repository,
-   use the ``bentoml pull`` CLI command followed by the Bento name and tag.
+* **Pull a Bento**: To fetch a specific version of a Bento from BentoCloud, run:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   bentoml pull <bento_name>:<tag>
+     bentoml pull <bento_name>:<tag>
 
-1. **Push a Bento:** Push a new Bento version to the repository
-   using the ``bentoml push`` command followed by the Bento name and tag.
+* **Push a Bento:** To upload a new version of an existing Bento to BentoCloud, run the following command. Note that you can push a Bento directly to BentoCloud without manually pushing the model packaged in the Bento. When you push a Bento to BentoCloud, the associated model is automatically uploaded.
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   bentoml push <bento_name>:<tag>
+     bentoml push <bento_name>:<tag>
 
-You can view your models in BentoCloud’s UI in `cloud.bentoml.com/bento_repositories <http://cloud.bentoml.com/bento_repositories>`_.
+The **Bentos** page displays all available Bentos on BentoCloud.
 
 .. image:: ../../_static/img/bentocloud/manage-bentos.gif
    :alt: manage-bentos.gif
-
-Python SDK
-==========
-
-In addition to CLI operations, BentoCloud also provides a Pythonic API for managing models and bentos.
-
-.. TODO::
-    Link Python API reference.
-
```

Comment:
Do we want to mention our client here?

Reply:
I think I will need more time to investigate the usage for it. But before that, I think it is better to remove it for now. I don't think we need to unveil sth in docs if it is not ready.